src/app/organization/organization.service.ts

```diff
@@ -44,7 +44,7 @@ export class OrganizationService {
    * @returns {Observable<any>} Employees data
    */
   getEmployees(): Observable<any> {
-    return this.http.get('/staff');
+    return this.http.get('/staff?status=all');
```

Comment:
```suggestion
    const httpParams = new HttpParams().set('status', 'all');
    return this.http.get('/staff', { params: httpParams });
```

The convention of `httpParams` has been followed in the entire web-app and will like to maintain the consistency further. Please change it accordingly so it can be merged.